### PR TITLE
Sisyphus component docs

### PR DESCRIPTION
**Pull request by jkeljo**
**Description:**
Adding documentation for the new Sisyphus component

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#14472

## Checklist:

- [X] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Looks great @jkeljo! Thanks a lot 🏅 
Can be merged as soon as the parent PR gets merged.

**Review by frenck — DISMISSED** [review]

**Review by frenck — DISMISSED** [review]

File: source/_components/light.sisyphus.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+description: "Instructions on how to integrate the light in your Sisyphus Kinetic Art Table into Home Assistant."
+date: 2018-05-06 22:26
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: sisyphus.png
+ha_category: Light
+ha_iot_class: "Local Push"
+ha_release: 0.74
```
Comment (MartinHjelmare) [inline source/_components/light.sisyphus.markdown]: 0.75

File: source/_components/media_player.sisyphus.markdown [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+description: "Instructions on how to integrate media playing on your Sisyphus Kinetic Art Table into Home Assistant."
+date: 2018-05-06 22:26
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: sisyphus.png
+ha_category: Light
+ha_iot_class: "Local Push"
+ha_release: 0.74
```
Comment (MartinHjelmare) [inline source/_components/media_player.sisyphus.markdown]: 0.75

File: source/_components/media_player.sisyphus.markdown [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+layout: page
+title: "Sisyphus Media Player"
+description: "Instructions on how to integrate media playing on your Sisyphus Kinetic Art Table into Home Assistant."
+date: 2018-05-06 22:26
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: sisyphus.png
+ha_category: Light
```
Comment (MartinHjelmare) [inline source/_components/media_player.sisyphus.markdown]: Media Player?

File: source/_components/sisyphus.markdown [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+description: "Instructions on how to integrate your Sisyphus Kinetic Art Table within Home Assistant."
+date: 2018-05-06 22:26
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: sisyphus.png
+ha_category: Hub
+featured: false
+ha_release: 0.74
```
Comment (MartinHjelmare) [inline source/_components/sisyphus.markdown]: 0.75

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**